ironfish/src/network/messages/newPooledTransactionHashes.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { TransactionHash } from '../../primitives/transaction'
+import { NetworkMessageType } from '../types'
+import { NetworkMessage } from './networkMessage'
+
+export class NewPooledTransactionHashes extends NetworkMessage {
+  hashes: TransactionHash[]
+
+  constructor(hashes: TransactionHash[]) {
+    super(NetworkMessageType.PooledTransactionsRequest)
```

Comment:
Should this be a different message type?
```suggestion
    super(NetworkMessageType.NewPooledTransactionHashes)
```

Reply:
yep, copy and paste error